docs/book/src/topics/multitenancy.md

```diff
@@ -87,7 +89,47 @@ data:
   password: PASSWORD
 ```
 
-## Manual Service Principal Identity
+### User-Assigned Managed Identity
+
+<aside class="note">
+
+<h1> Note </h1>
```

Comment:
Is this supposed to be a <h1> heading? I'm guessing it renders differently on the actual webpage but it does look really big in the GitHub preview.

Reply:
https://deploy-preview-2371--kubernetes-sigs-cluster-api-provider-azure.netlify.app/topics/multitenancy.html#note it renders nicely in the book website, it's meant to be the note heading